config/locales/validations/sales/sale_information.en.yml

```diff
@@ -0,0 +1,125 @@
+en:
+  validations:
+    sales:
+      sale_information:
+        ownershipsch:
+          discounted_ownership_value: "The mortgage%{mortgage}%{deposit_and_grant_sentence} added together is %{mortgage_deposit_and_grant_total}.</br></br>The full purchase price%{discount_sentence} is %{value_with_discount}.</br></br>These two amounts should be the same."
+          outright_sale_value: "The mortgage%{mortgage} and cash deposit (%{deposit}) when added together is %{mortgage_and_deposit_total}.</br></br>The full purchase price is %{value}.</br></br>These two amounts should be the same."
```

Comment:
Do we want to adjust how we're passing mortgage in here to be more like we're doing with deposit? It seems a bit inconsistent for editing.

Reply:
I think in some cases mortgage is not used or 0, but the copy for error messages was the same, if we did it the same as deposit, in some cases it would be empty brackets